contracts/cw-blob/src/lib.rs

```diff
@@ -58,37 +58,38 @@ pub mod interface {
                 .checksum()
                 .unwrap()
         }
+    }
 
+    pub trait MigrationFromBlob<Chain: CwEnv>:
```

Comment:
```suggestion
    pub trait DeterministicInstantiation <Chain: CwEnv>:
```

"blob" doesn't really say anything and devs don't really have to know that it instantiates a blob-contract first.